planner/core/optimizer.go

```diff
@@ -386,6 +387,98 @@ func postOptimize(sctx sessionctx.Context, plan PhysicalPlan) PhysicalPlan {
 	return plan
 }
 
+// prunePhysicalColumns currently only work for MPP(HashJoin<-Exchange).
```

Comment:
We can add some comments that this func only solves this case since the exchange will send extra data via the network. We add it to avoid unexpected network costs. Help others know why we need this code segment.

Reply:
Done.